packages/scss/src/commons/config.scss

```diff
@@ -96,120 +93,103 @@ $error: (
 	700: #da2f2f,
 	800: #aa0e0e,
 	900: #630303,
-) !default;
-
-// Colors : Neutral
-
-$grey: (
-	text: #ffffff,
-	25: #f3f6fc,
-	50: #e7edf9,
-	100: #dbe3f5,
-	200: #ced9ee,
-	300: #becbe4,
-	400: #acbbd7,
-	500: #96a6c5,
-	600: #667799,
-	700: #445473,
-	800: #2a3551,
-	900: #0b1732,
-) !default;
+);
 
-// Colors : Decoratives
```

Comment:
Shouldn't these colors be outside of the commons? I don't think that many softs are currently using them.

Reply:
I've modified the structure a little so that you can configure LF like this and reduce the CSS variables: 

```
@use '@lucca-front/scss/src/commons/config' with (
	$newPalettes: [],
	$decoratives: [],
);
```

Reply:
Nice! Thanks